server/templates/static/datasets.html

```diff
@@ -19,431 +19,1276 @@
-  </p> -->
-
-  <h2>
-    <a href="https://chronicdata.cdc.gov/500-Cities/500-Cities-Local-Data-for-Better-Health-2018-relea/6vp6-wxuq">
-      CDC - 500 Cities: Local Data for Better Health
-    </a>
-  </h2>
-  <p>
-    The 500 Cities Project datasets contain model-based small area estimates for 27 measures of chronic
-    disease related to unhealthy behaviors (5), health outcomes (13), and use of preventive services
-    (9).
-    <br>
-    <br>
-    Data made available under <a href="https://www.cdc.gov/other/agencymaterials.html">CDC Data Terms of Service</a>.
-  </p>
+        <br />
+        <h3><a href="https://nces.ed.gov/programs/edge/Demographic/ACS">Census Bureau - American Community Survey
+            Education
+            Tabulation (ACS-ED)</a></h3>
+        <p> The National Center for Education Statistics collaborates with the US Census Bureau to create a variety of
+          custom
+          American Community Survey (ACS) data files that describe the condition of school-age children in the United
+          States at
+          the country, state, and school district level. ACS-ED is updated annually based on ACS five-year period
+          estimates.
+          Data Commons includes statistics on dozens of variables used in the ACS-ED District Demographic Dashboard.
+        </p>
+        <p> Data made available under the public domain.
+        </p>
 
-  <h2>
-    <a href="https://data.cdc.gov/Environmental-Health-Toxicology/Daily-County-Level-PM2-5-Concentrations-2001-2014/qjju-smys/data">
-      CDC Wonder - Daily County-Level PM2.5 Concentrations
-    </a>
-  </h2>
-  <p>
-    Daily County-Level PM2.5 Concentrations "provides modeled predictions of particulate matter
-    (PM2.5) levels from the EPA's Downscaler model. These data are used by the CDC's National
-    Environmental Public Health Tracking Network to generate air quality measures. Data are at the
-    county levels for 2001-2014".
-    <br>
-    <br>
-    Data made available under <a href="https://wonder.cdc.gov/datause.html">CDC Wonder Data Terms of Service</a>.
-  </p>
+        <br />
+        <h3><a href="https://www.census.gov/programs-surveys/geography/geographies/mapping-files.html">Census Bureau -
+            Cartographic Boundary Files</a></h3>
+        <p> Data Commons has KML files from <a
+            href="https://www.census.gov/geographies/mapping-files/time-series/geo/kml-cartographic-boundary-files.2018.html">2018
+            US Cartographic Boundaries</a> for the following place types:
+        </p>
+        <ul>
+          <li>Congressional Districts: 116th Congress
+          <li>County
+          <li>Place
+          <li>State
+          <li>Census Tracts
+          <li>County Subdivisions
+          <li>ZIP Code Tabulation Areas
+          <li>School Districts - Elementary
+          <li>School Districts - Secondary
+          <li>School Districts - Unified</li>
```

Comment:
please close all < li > tags or none (either works, but we should be consistent)

Reply:
wasn't sure if you meant consistent per ul, so I added `</li>` to all` <li>` in the page.